src/oas3/transformers/parameters.ts

```diff
@@ -49,10 +53,24 @@ export const translateToSharedParameters = withContext<
     if (isReferenceObject(value)) {
       // note that unlike schemas, we don't handle proxy $refs here
       // we need resolved content to be able to determine the kind of parameter to push it to the correct array
-      this.references[`#/components/parameters/${key}`] = {
-        resolved: false,
-        value: value.$ref,
-      };
+      if (value.$ref.startsWith('#')) {
+        // a reference WITHIN this http-spec document
+        this.references[`#/components/parameters/${key}`] = {
+          resolved: false,
+          value: value.$ref,
+        };
+      } else {
+        // a reference to OUTSIDE this http-spec document
+        this.references[`#/components/parameters/${key}`] = {
+          resolved: true,
+          value: `#/components/unknownParameters/${sharedParameters.unknownParameters.length}`,
+        };
+        // We have to add this to shared parameters here, rather than
```

Comment:
```suggestion
        // We have to add this to shared parameters here, rather than in the loop below
```
I guess is what you meant?